setup.py

```diff
@@ -30,6 +30,7 @@
         "python-dateutil>=2.7.3",
         "PyYAML>=3.13",
         "rfc3986>=1.2.0",
```

Comment:
LGTM one caveat - I would add pyinstrimenr under extras_required as “profiling”. You may need to them try catch importing it in the code.

Reply:
@adamhadani I am not too familiar with `extras_require`, but I gave it a shot.